src/JSON For mIRC.mrc

```diff
@@ -570,7 +570,7 @@ alias JSONClose {
   }
 
   ;; Validate @Name
-  elseif (: isin $1) && (w isincs %Switches || JSON:* !iswmcs $1) {
+  elseif (: isin $1) && (w isincs %Switches) || (JSON:* !iswmcs $1) {
```

Comment:
This is incorrect. The correct version would be the following but I opted not to have duplicate `()`:

```
elseif (: isin $1) && ((w isincs %Switches) || (JSON:* !iswmcs $1)) {
```